scraper.rb

```diff
@@ -20,7 +20,13 @@ def tidy
 def scrape_list_page(url)
   page = MembersPage.new(response: Scraped::Request.new(url: url).response)
   page.members.each do |member|
-    ScraperWiki.save_sqlite([:id, :term], member.to_h) unless member.name == 'Vaccant Poste'
+    arabic_member_page = MemberPage.new(
+      response: Scraped::Request.new(
+        url: member.source.sub('/en/', '/ar/')
+      ).response
+    )
+    data = member.to_h.merge(name__ar: arabic_member_page.name)
+    ScraperWiki.save_sqlite([:id, :term], data) unless member.name == 'Vaccant Poste'
```

Comment:
There doesn't seem to be much value in loading the arabic page in the "Vaccant Poste" case — this is just doing quite a lot of extra work only to throw it away.

I suspect you might want to reject those earlier, e.g:

```ruby
  page.members.reject { |m| m.name == 'Vaccant Poste' }.each do |member|
```